test/test_dynamic_shapes.py

```diff
@@ -4,6 +4,7 @@
 import torch, torch_xla
 import torch_xla.core.xla_model as xm
 
+pd = torch._C._EnablePythonDispatcher()
```

Comment:
Do we want to enable python dispatcher for master?

Reply:
Please correct me if I'm wrong. To my understanding, for dynamic shape feature to run, we enable python dispatch by default.